File: src/main/kotlin/com/tft/guide/repository/SynergyRepository.java
package com.tft.guide.repository;

import com.tft.guide.entity.Synergy;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.data.querydsl.QuerydslPredicateExecutor;

public interface SynergyRepository extends MongoRepository<Synergy, String> {
}

File: src/main/kotlin/com/tft/guide/repository/SynergyRepository.kt
package com.tft.guide.repository

import com.tft.guide.entity.Item
import com.tft.guide.entity.Synergy
import org.springframework.data.mongodb.repository.MongoRepository

interface SynergyRepository : MongoRepository<Synergy, String> {
    fun findAllBySeason(season: String): List<Synergy>
}